js/admin/ppom-meta-table.js

```diff
@@ -52,12 +44,22 @@ jQuery(function($){
 				$.post(ajaxurl, data, function(resp){
 					$('#ppom_delete_selected_products_btn').html('Delete');
 					if (resp) {
-						swal.fire({title: "Done", text: resp, type: "success" ,confirmButtonColor: '#217ac8'}).then(()=>location.reload());
-					}else{
-						swal.fire(resp, "", "error").then();
+						window?.ppomPopup?.open({
+							title: window?.ppom_vars?.i18n.popup.finishTitle,
+							hideCloseBtn: true,
+							onConfirmation: () => location.reload(),
+							onClose: () => location.reload()
+						});
+					} else {
+						window?.ppomPopup?.open({
+							title: window.ppom_vars.i18n.popup.errorTitle,
```

Comment:
You're not using `window?.ppom_vars?` here. Any reason why?

Reply:
WordPress will not load the script unless the global object is also loaded (in this case, `ppom_vars`).